packages/steiger-plugin-fsd/src/import-locality/index.spec.ts

```diff
@@ -6,7 +6,9 @@ import importLocality from './index.js'
 vi.mock('tsconfck', async (importOriginal) => {
   return {
     ...(await importOriginal<typeof import('tsconfck')>()),
-    parse: vi.fn(() => Promise.resolve({ tsconfig: { compilerOptions: { paths: { '@/*': ['/*'] } } } })),
+    parse: vi.fn(() =>
+      Promise.resolve({ tsconfig: { compilerOptions: { paths: { '@/*': ['/users/user/project/src/*'] } } } }),
```

Comment:
note: but here, for instance, it's fine because the alias was already an absolute path, so it's okay to just replace it